.gitignore

```diff
@@ -57,3 +57,6 @@ dkms.conf
 
 # Build
 build/*
+
+# Tsim
+tsim-leon3
```

Comment:
Why is this in gitignore? If tsim was used, I guess it shoul be installed in a dedicated directory and added to path...

Reply:
ack